.pre-commit-config.yaml

```diff
@@ -1,7 +1,7 @@
 repos:
--   repo: https://github.com/ambv/black
+  - repo: https://github.com/ambv/black
     rev: stable
     hooks:
-    - id: black
+      - id: black
 default_language_version:
-  python: python3.7
+  python: python3.11
```

Comment:
I'd love your thoughts on this: should we wait to upgrade the Python version in another PR where we can do a full upgrade e.g. update the version of Python used in CI when the tests are run, and update the metadata in `setup.py` to reflect that version?

Reply:
Oh, nevermind! I just saw https://github.com/lifeomic/phc-sdk-py/pull/188, so looks like that has already been addressed.